proposals/init.md

```diff
@@ -0,0 +1,582 @@
+Init Only Members
+=====
+
+## Summary
+This proposal adds the concept of init only properties to C#. These properties
+can be set at the point of object creation but become effectively `get` only 
+once object creation has completed. This allows for a much more flexible
+immutable model in C#. 
+
+## Motivation
+The underlying mechanisms for building immutable data in C# haven't changed
+since 1.0. They remain:
+
+1. Declaring fields as `readonly`.
+1. Declaring properties that contain only a `get` accessor.
```

Comment:
```suggestion
2. Declaring properties that contain only a `get` accessor.
```

Reply:
[As per above](https://github.com/dotnet/csharplang/pull/3367#discussion_r411008323) this is intentional.